tests/e2e/createBand.spec.ts

```
@@ -0,0 +1,28 @@
+    await login(page);
+
+    const { name: bandName } = await createBand(page);
+
+    const actualBandName = await page.textContent('text=Test Band');
+    expect(actualBandName).toBe(bandName);
+  });
+
+  test('should delete a band', async ({ page }) => {
+    await login(page);
+
+    const { name: bandName, url: bandUrl } = await createBand(page, 'Band To Delete');
```

ESLint

[error] 'bandName' is assigned a value but never used.

(no-unused-vars)